app/models/relationship.rb

```diff
@@ -3,4 +3,6 @@ class Relationship < ApplicationRecord
   belongs_to :followed, class_name: 'User'
   validates :follower_id, presence: true
   validates :followed_id, presence: true
+
+  scope :find_relationship, ->(follower_id, followed_id) { where(follower_id: follower_id, followed_id: followed_id) }
```

Comment:
https://stackoverflow.com/questions/31329554/find-by-inside-a-scope-is-firing-2-queries
Ở đây có đề cập đến việc find_by thực sự tạo một truy vấn nên k nên dùng trong scope.

Reply:
mà nên dùng (where).first

Reply:
ok em